src/deprecated.jl

```diff
@@ -460,10 +460,10 @@ for allowcomments in tf, skipblanks in tf, allowescapes in tf, wsv in tf
     end
 end
 
-function bytematch{T <: String}(bytes::Vector{UInt8},
-                                    left::Integer,
-                                    right::Integer,
-                                    exemplars::Vector{T})
+function bytematch(bytes::Vector{UInt8},
+                       left::Integer,
```

Comment:
Incorrect indentation. BTW, no need to define `T`.

Reply:
Still incorrect indentation.

Reply:
What's the proper indentation?
```
function func(obj::Vector{T,R},
    left::T,
    right::T) where {T <: Any,
    R <: Any}
```
Always four spaces for long lines?
The other functions in the package have it running past 80 characters and matching previous line start position.

Reply:
As for all functions in this project, align arguments vertically with the first argument. Align type parameters with the first one.

Reply:
Thanks!